### DISCUSSION TITLE
2 or more  Nested groups layout dynamically inject in  PanelGroup  then resizable-panels not work.

### DISCUSSION THREAD

**Discussion by eshak-cdda** [bvaughn/react-resizable-panels#314] [Q&A] [answered]
I want to make every panel draggable and resizable. ist work 1 layer. if this is nested then it does not work. nested one changing then the upper one is changing her height and width.

**Comment by bvaughn** [comment, marked as answer]
Sorry, I'm not really sure what you're describing.

Nested groups are supported. You can see an example of how to write the code here:

https://github.com/bvaughn/react-resizable-panels/blob/44d3a65d649dd943168e581fe400c774cfc06cfc/packages/react-resizable-panels-website/src/routes/examples/Nested.tsx#L24-L58

A demo of that code can be seen here:

https://react-resizable-panels.vercel.app/examples/nested

**Reply by eshak-cdda** [reply]
Thank you for your response. now we have resolved our problems. we combine this with React DND. Thank you again.
